configure-windows-visual-studio.bat

```diff
@@ -24,11 +28,14 @@ if "%ERRORLEVEL%" NEQ "0" exit /B 1
 if "%VS_VERSION%"=="" echo Visual studio version was not provided. Exiting & exit /B 1
 if "%PTHREAD_HOME%"=="" echo pthread-win32 directory was not provided. Exiting & exit /B 1
 if "%PCAP_SDK_HOME%"=="" echo WinPcap/Npcap SDK directory was not provided. Exiting & exit /B 1
+if "%ZSTD_HOME%"=="" echo WinPcap/Npcap SDK directory was not provided. ZSTD Support will be skipped!
```

Comment:
did you mean to say "ZStd SDK directory was not provided"?

Reply:
Yes, fixed